index.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const teams = require('./teams.json')
+
+const app = express()
+
+app.get('/teams', (request, response) => {
+    response.send(teams)
+})
+
+app.get('/teams/:id', (request, response) => {
+    let abbreviationId
+    let teamId
```

Comment:
These are strange names for lists of teams. `abbreviationId` and `teamId` suggest an identifier that points to a specific abbreviation or team when in fact these variables hold a list of teams.